### PR TITLE
fixup! Show and hide menu windows once visibility is changed (#52)

### PR DESCRIPTION
**Pull request by msisov**
Bugs fixing:
    
      Hide method should be implemented separately from visibility.
      Otherwise some ServerWindows, which belong to main window,
      might become hidden, which triggers hiding native window and
      nothing is shown. For example, when one clicks "Downloads" in the
      menu, the main window's one ServerWindow becomes visible and another
      one is invisible, which makes the native window for those ServerWindows
      to be invisible.
    
      Second thing is that event_dispatcher must set delegate_->SetNativeCapture.
      I've removed that change in this CL.

### CODE REVIEW

File: ui/aura/mus/window_tree_host_mus.cc [hunk 1]
```diff
@@ -188,6 +188,10 @@ display::Display WindowTreeHostMus::GetDisplay() const {
 void WindowTreeHostMus::HideImpl() {
   WindowTreeHostPlatform::HideImpl();
   window()->Hide();
+
+  WindowPortMus* window_port = WindowPortMus::Get(window());
+  if (window_port)
+	  window_port->OnWillHideNativeWindow();
```
Comment (tonikitoo) [inline ui/aura/mus/window_tree_host_mus.cc]: nit: indentation

Comment (msisov) [reply ui/aura/mus/window_tree_host_mus.cc]: done!

**Comment by tonikitoo** [comment]
ok, lets give this a try, although I fear for it to be a rebase problem every week, but time will tell us.

quick plan:

- finish this week's rebase
- rebase this PR against it
- do a final review pass and push next.

**Comment by msisov** [comment]
Please continue reviewing. Rebase pushed, the PR is rebased.

**Review by tonikitoo — APPROVED** [review]
lgtm

**Pull request merged**